shotover-proxy/Cargo.toml

```diff
@@ -14,7 +14,7 @@ alpha-transforms = []
 
 [dependencies]
 pretty-hex = "0.3.0"
-hex = "0.4.3"
+hex = { version = "0.4.3", features = ["serde"] }
```

Comment:
I dont think we make use of this serde feature anywhere?
Seems to enable this module that we dont use: https://docs.rs/hex/0.4.3/hex/serde/index.html
That module sure is interesting though, does it let us avoid allocating the Vec<u8> in between?

Reply:
True, I must've left it in there trying something else and assumed I still needed it.